src/legacy/core_plugins/console/public/np_ready/application/models/legacy_core_editor/legacy_core_editor.ts

```diff
@@ -297,30 +297,30 @@ export class LegacyCoreEditor implements CoreEditor {
       // pageY is relative to page, so subtract the offset
       // from pageY to get the new top value
       const offsetFromPage = $(this.editor.container).offset()!.top;
-      const startRow = range.start.lineNumber - 1;
+      const startLine = range.start.lineNumber;
```

Comment:
nit: at first glance, i don't think the const name `startLine` is clear in how it differs from `firstLine`

Reply:
After this https://github.com/elastic/kibana/pull/52270 was merged a distinction was created between row (zero indexed) and line (not-zero indexed). This was informed by the approach taken by Monaco 😅